src/utils/session.ts

```diff
@@ -1,11 +1,13 @@
+import { Session } from "@companieshouse/node-session-handler";
 import { Request } from "express";
 import { isDetails } from "private-api-sdk-node/dist/services/presenter-account/types";
 import { type Details } from "private-api-sdk-node/src/services/presenter-account/types";
 
 export const PRESENTER_ACCOUNT_SESSION_KEY = "presenter_account_details";
 
 export function getPresenterAccountDetails(req: Request): Details {
-    const presenterAccountDetails = req.session?.getExtraData(PRESENTER_ACCOUNT_SESSION_KEY);
+    const session = req.session as Session;
+    const presenterAccountDetails = session.getExtraData(PRESENTER_ACCOUNT_SESSION_KEY);
```

Comment:
The session is optional and so it shouldn't just be cast to a Session type.
If the session is undefined that needs to be handled. Or at the very least have a descriptive error logged out.